components/PlantCard.js

```diff
@@ -1,26 +1,33 @@
 import Image from "next/image";
+import Link from "next/link";
 import styled from "styled-components";
 
 const Card = styled.article`
   position: relative;
   margin-top: 20px;
   height: 55vw;
   width: 90vw;
-  border: 0px solid;
+  border: 0 solid;
```

Comment:
Hint: if the border has a thickness of 0, the border is not displayed. Every further border styling is therefore redundant, so you can just remove the whole border style (Remove  the complete line 10).